llpc/lower/llpcSpirvProcessGpuRtLibrary.cpp

```diff
@@ -125,17 +125,26 @@ void SpirvProcessGpuRtLibrary::processLibraryFunction(Function *&func) {
   const StringRef fetchTrianglePositionFromRayQueryFuncName =
       m_context->getPipelineContext()->getRayTracingFunctionName(Vkgc::RT_ENTRY_FETCH_HIT_TRIANGLE_FROM_RAY_QUERY);
 
+  const StringRef rayQueryGetInstanceIndx =
+      m_context->getPipelineContext()->getRayTracingFunctionName(Vkgc::RT_ENTRY_INSTANCE_INDEX);
+
+  const StringRef rayQueryGetInstanceId =
+      m_context->getPipelineContext()->getRayTracingFunctionName(Vkgc::RT_ENTRY_INSTANCE_ID);
```

Comment:
These are not specific for ray query, maybe: `getInstanceIdFuncName` and `getInstanceIndexFuncName`.

Reply:
Done.